src/data/footer.js

```diff
@@ -0,0 +1,32 @@
+const arr = [
```

Comment:
an intuitive variable name might be better

Reply:
so **arr** can be renamed as **footer_data**